addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -140,7 +140,6 @@ def res_partner_get(self, email=None, name=None, partner_id=None, **kwargs):
         versions of the mail plugin but necessary for supporting older versions, only the route name is deprecated not
```

Comment:
Sugg for the commit message:

> When an email is opened and is from the default notification mechanism,
i.e: coming from something like "notifications@mail.odoo.com", we don't want to
try to match the email address to an existing contact partner.
> 
> Instead, we now display a custom message on the mail addon side mentioning to
search the contact manually.